### ISSUE TITLE
Page order messed up

### ISSUE THREAD

**Issue by SuperNici** [7USTIN/manga-cli#6]
# Steps to reproduce
1. `manga-cli`
2. search for `chainsaw man`
3. Select chapter 1

Now if you compare it to [the website](https://readmanganato.com/manga-dn980422/chapter-1) you'll see that the order is messed up. Page 50-53 are supposed to be in the beginning. Page 12, 23, 34 as well to name a few.

This is just an example but the issue is not limited to just this manga.

# Possible solution

@stl3 mentioned the following in issue #5 which also in a way applies to this.
> I think perhaps `convert_imgs_to_pdf` needs something to guarantee the images are being fed sequentially (I don't actually know if convert does that by default).

**Comment by stl3** [comment]
Ok, after some testing, it seems that all we need to do is make sure there are leading 0's on the *.jpgs - so like 001.jpg, 002.jpg.
I tested both with imagemagick and img2pdf and the page sequences were correct but I use some copy pasta old code I found:
```
# Make pictures sequential
                find "${image_dir}" -type f -name '*.jpg' -exec bash -c '
                    for pathname do
                    dirpath=${pathname%/*}  # or: dirpath=$(dirname "$pathname")
                    name=${pathname##*/}    # or: name=$(basename "$pathname")
                        number=${name##*_}
                        number=${number%.jpg}
                    printf -v newname "%s_%.3d.jpg" seq "$number"
        mv "$pathname" "$dirpath/$newname"
    done' bash {} +

```
Only use this code for testing though, there are better ways to do this lol. Paste it above this line:
`convert "${image_dir}/*.jpg" "${pdf_dir}"`

I've only compared it against how the website was displaying this manga, but it looks right so far.

**Comment by 7USTIN** [comment]
Thanks again, @supernici and @stl3!

I just pushed a commit that should fix this issue and issue #5. I have no idea how I didn't notice this earlier lol.

I fixed the problem by storing the sorted output of `ls -v` in an array. Then just adding the image directory in front of the filenames and checking if the prefix `python3 -m` is needed, in case you have img2pdf with pip3 installed.

**Comment by stl3** [comment]
I know this isn't needed atm but I found an additional short line that might be useful in case anyone wanted to try this sorting method in the future with img2pdf:
`find ${image_dir} -name '*.jpg' -print0 | sort -z -V | xargs -0r img2pdf -o "${pdf_dir}"`
